src/demo/java/org/junitpioneer/jupiter/DefaultLocaleTimezoneExtensionDemo.java

```diff
@@ -19,6 +19,18 @@
 
 public class DefaultLocaleTimezoneExtensionDemo {
 
+	protected static Locale createLocale(String language, String country, String variant) {
```

Comment:
Move this functionality to `PioneerUtils` and use it consistently across the extension and tests, IMO.

Reply:
Moved 👍